views/admin-page-settings.php

```diff
@@ -50,6 +50,61 @@
 			</div>
 		</div>
 
+		<div class="prpl-column">
```

Comment:
Until we implement licensing, perhaps this should be hidden? 

```suggestion
		<div class="prpl-column" style="display:none;">
```

Reply:
Done